uv_copy_and_paste_uv/cpuv_transfer_uv_operation.py

```diff
@@ -178,7 +198,7 @@ def main_parse(self, active_obj, bm, uv_layer):
 
     else:
         self.report({'WARNING'}, "Two faces should should share one edge!!")
-        return {'CANCELLED'}
+        return None
```

Comment:
Why return None?

If `main_parse` return `None`, does operation continue not error? (ex. L117)